README.md

```diff
@@ -1,368 +1,186 @@
-| ---- | ----------- | -------- |
-|`-i`, `--input`| Input path of the `.i3dm`| :white_check_mark: Yes |
-|`-o`, `--output`| Output path of the resulting `.glb` | No |
-|`-f`, `--force`|Overwrite output file if it exists.| No, default `false` |
-
-### cmptToGlb
+#### cmptToGlb
 
-Extracts the glb models from a cmpt tile. If multiple models are found a number will be appended to the
-output file name. Since this tool does not process an entire tileset, it cannot be used with the Pipeline tool.
+Extracts the glb models from a cmpt tile. If multiple models are found a number will be appended to the output file name.
 
 ```
-node ./bin/3d-tiles-tools.js cmptToGlb -i ./specs/data/composite.cmpt -o ./output/extracted.glb
+npx ts-node ./src/main.ts b3dmToGlb -i ./specs/data/composite.cmpt -o ./output/extracted.glb
 ```
 
-| Flag | Description | Required |
-| ---- | ----------- | -------- |
-|`-i`, `--input`| Input path of the `.cmpt`| :white_check_mark: Yes |
-|`-o`, `--output`| Output path of the resulting `.glb` | No |
-|`-f`, `--force`|Overwrite output file if it exists.| No, default `false` |
-
-### optimizeB3dm
+#### optimizeB3dm
 
-Optimize a b3dm using [gltf-pipeline](https://github.com/CesiumGS/gltf-pipeline/blob/main/README.md). Since this tool does not
-process an entire tileset, it cannot be used with the Pipeline tool.
+Optimize a b3dm using [gltf-pipeline](https://github.com/CesiumGS/gltf-pipeline/blob/main/README.md). 
 
 ```
-node ./bin/3d-tiles-tools.js optimizeB3dm -i ./specs/data/batchedWithBatchTableBinary.b3dm -o ./output/optimized.b3dm
+npx ts-node ./src/main.ts optimizeB3dm -i ./specs/data/batchedWithBatchTableBinary.b3dm -o ./output/optimized.b3dm
 ```
 
-Quantize floating-point attributes and oct-encode normals
-```
-node ./bin/3d-tiles-tools.js optimizeB3dm -i ./specs/data/batchedWithBatchTableBinary.b3dm -o ./output/optimized.b3dm --options -q -n
-```
+Additional command line options:
 
 | Flag | Description | Required |
 | ---- | ----------- | -------- |
-|`-i`, `--input`| Input path of the `.b3dm`| :white_check_mark: Yes |
-|`-o`, `--output`| Output path of the resulting `.b3dm` | No |
-|`-f`, `--force`|Overwrite output file if it exists.| No, default `false` |
 |`--options`|All arguments past this flag are consumed by gltf-pipeline.| No |
 
+**Examples**: 
+
+Quantize floating-point attributes and oct-encode normals:
+```
+npx ts-node ./src/main.ts optimizeB3dm -i ./specs/data/batchedWithBatchTableBinary.b3dm -o ./output/optimized.b3dm --options -q -n
+```
+
 To use tileset texture compression, pass the [`texcomp` flags](https://github.com/CesiumGS/gltf-pipeline/blob/main/README.md#command-line-flags)
 ```
 node ./bin/3d-tiles-tools.js optimizeB3dm -i ./specs/data/Textured/batchedTextured.b3dm -o ./output/optimized.b3dm --options --texcomp.dxt1.enable --texcomp.dxt1.quality=5 --texcomp.etc1.enable
 ```
 This example optimizes the b3dm and compresses the textures into `dxt1` and `etc1` formats.
```

Comment:
I changed that to be a draco example, but also noticed that there had been some issues with passing the options to `gltf-pipeline`: The old version exposed a `parseArguments` function that was applied to the "raw" `string[]` to obtain the actual `options` object. I now tried to "emulate" that, with some quirks (the name is `draco.foo` at the CLI, but `dracoOptions.foo` in the actual `options` object...).

In a test with the example that is now in the README, 
```
npx ts-node ./src/main.ts optimizeB3dm -i ./specs/data/Textured/batchedTextured.b3dm -o ./output/optimized.b3dm --options --draco.compressMeshes --draco.compressionLevel=9
```
the output indeed was a B3DM where the GLB contained the Draco part, so that seems to work now...